editor/reducer.js

```diff
@@ -306,38 +306,44 @@ export function isTyping( state = false, action ) {
  * @param  {Object} action Dispatched action
  * @return {Object}        Updated state
  */
-export function blockSelection( state = { start: null, end: null, focus: null }, action ) {
+export function blockSelection( state = { start: null, end: null, focus: null, isMultiSelecting: false }, action ) {
```

Comment:
Would it be time to refactor `blockSelection` as a combined reducer of all its keys? That design would seem a little less error-prone, but that may be just my perception.

Reply:
That make sense. Let's attempt that in a separate PR.